packages/shared/src/components/Feed.tsx

```diff
@@ -182,6 +183,7 @@ export default function Feed<T>({
         adPostLength: isSquadFeed ? 2 : undefined,
         showAcquisitionForm,
         ...(showMarketingCta && { marketingCta }),
+        ...(plusEntryFeed && { plusEntry: plusEntryFeed }),
```

Comment:
Since feed has multiple pages, just making sure it is not injected into each one?